buildSrc/src/main/java/edu/wpi/gripgenerator/FileParser.java

```diff
@@ -216,43 +224,79 @@ public Expression getDefaultValue(String defaultValue) {
         compilationUnits.putAll(enumVisitor.generateCompilationUnits());
 
         DefinedMethodCollection collection = new DefinedMethodCollection(baseClassName,
-                new DefinedMethod("add", false, "Mat", "Mat", "Mat"),
-                new DefinedMethod("subtract", false, "Mat", "Mat", "Mat").addDescription("Calculates the per-pixel difference between two images"),
-                new DefinedMethod("multiply", false, "Mat", "Mat", "Mat"),
-                new DefinedMethod("divide", false, "Mat", "Mat", "Mat"),
-                new DefinedMethod("scaleAdd", false, "Mat", "double", "Mat", "Mat"),
-                new DefinedMethod("normalize", false, "Mat", "Mat"),
-                new DefinedMethod("batchDistance", false, "Mat", "Mat"),
-                new DefinedMethod("addWeighted", false, "Mat"),
+                new DefinedMethod("add", false, "Mat", "Mat", "Mat")
+                        .addDescription("Calculates the per-pixel sum of two Mats"),
+                new DefinedMethod("subtract", false, "Mat", "Mat", "Mat")
+                        .addDescription("Calculates the per-pixel difference between two Mats"),
+                new DefinedMethod("multiply", false, "Mat", "Mat", "Mat")
+                        .addDescription("Calculates the per-pixel scaled product of two Mats"),
+                new DefinedMethod("divide", false, "Mat", "Mat", "Mat")
+                        .addDescription("Performs per-pixel division of two Mats"),
+                new DefinedMethod("bitwise_and", false, "Mat", "Mat")
+                        .addDescription("Computes bitwise conjunction of the two Mats"),
+                new DefinedMethod("bitwise_or", false, "Mat", "Mat")
+                        .addDescription("Calculates the per-element bit-wise disjunction of two Mats"),
+                new DefinedMethod("bitwise_xor", false, "Mat", "Mat")
+                        .addDescription("Calculates the per-element bit-wise \\\"exclusive or\\\" operation on two Mats"),
+                new DefinedMethod("bitwise_not", false, "Mat", "Mat")
+                        .addDescription("Calculates per-element bit-wise inversion of the input Mat")
+        )
+                .setDirectionDefaults(DefinedParamType.DefinedParamDirection.INPUT_AND_OUTPUT, "dst")
+                .setIgnoreDefaults("dtype");
+
+        DefinedMethodCollection collection2 = new DefinedMethodCollection(baseClassName,
```

Comment:
This allows you to take full advantage of the mask parameter as it was intended to be used.